.github/workflows/Unit-Test.yml

```diff
@@ -28,9 +28,12 @@ jobs:
             echo "ALL_CHANGED FILES is/are ${ALL_CHANGED_FILES}." && \
             export CHANGED_PACKAGES="$(echo ${ALL_CHANGED_FILES} | sed -e 's,packages/,,g' -e 's,\.rb,,g' | sort)" && \
             echo "CHANGED PACKAGES is/are ${CHANGED_PACKAGES}." && \
-            export X86_64_PACKAGES="$(for p in ${CHANGED_PACKAGES}; do grep -q "compatibility.*all\|x86_64" packages/$p.rb && echo $p; done | xargs)" && \
+            export SAME_GLIBC_PACKAGES="$(for i in ${CHANGED_PACKAGES} ; do if ! grep -q min_glibc packages/$i.rb; then echo $i ; else grep min_glibc packages/$i.rb | tr -d \"'\" | awk '{exit $2 < 2.27}' || echo $i ; fi ; done | xargs)" && \
+            echo "PR #${PR_NUMBER} has these Glibc 2.27 compatible packages: ${SAME_GLIBC_PACKAGES}" && \
```

Comment:
2.27 would be the lower glibc version, right?  If we set `min_glibc` in the package, it usually means a glibc version higher than 2.27 is needed.

Reply:
Awk's exit code is the reverse of most unix commands...

Reply:
That is fine.  I'm just mentioning the message should say `Glibc 2.27 incompatible` instead of compatible.